### PR TITLE
fix #402

### LINKED ISSUE #402: uniform vs. logUniform prior
so far we only allow for uniform, however, we should in my opinion distinguish between uniform and logUniform. in most cases we use actually log-uniform as we consider a uniform distribution on log scale.

### PR DESCRIPTION
**Pull request by paulstapor**
At least as far as possible with the current format, #402 is addressed.

### CODE REVIEW

File: petab/C.py [hunk 1]
```diff
@@ -93,17 +93,16 @@
...
-PARAMETER_SCALE_NORMAL = 'parameterScaleNormal'
 LAPLACE = 'laplace'
-PARAMETER_SCALE_LAPLACE = 'parameterScaleLaplace'
 LOG_NORMAL = 'logNormal'
 LOG_LAPLACE = 'logLaplace'
 
 PRIOR_TYPES = [
-    UNIFORM, NORMAL, LAPLACE, LOG_NORMAL, LOG_LAPLACE,
-    PARAMETER_SCALE_UNIFORM, PARAMETER_SCALE_NORMAL, PARAMETER_SCALE_LAPLACE]
+    UNINFORMATIVE, UNIFORM, NORMAL, LAPLACE,
```
Comment (dweindl) [inline petab/C.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=5243799)

File: petab/sampling.py [hunk 2]
```diff
@@ -42,13 +42,17 @@ def clip_to_bounds(x: np.array):
         return x
 
     # define lambda functions for each parameter
-    if p_type == UNIFORM:
+    if p_type == UNINFORMATIVE:
+        sp = (p_params[1] - p_params[0]) * np.random.random((n_starts,
+                                                             )) + p_params[0]
```
Comment (yannikschaelte) [inline petab/sampling.py]: autoformat gone crazy? start new line before

Comment (yannikschaelte) [reply petab/sampling.py]: i know it was the same ugly beforehand already.

**Comment by paulstapor** [comment]
taht was just about 5 seconds too late, sorry... :(

**Pull request merged**